scripts/build-testing-sdk.js

```diff
@@ -0,0 +1,40 @@
+/**
```

Comment:
Hey I'd like to hear why you think this way?

I chose not to do this because editors will loose syntax highlighting in some cases it seems. For instance previously when I looked at `scripts/build` the entire script was grey.

I'm not 100% I see an advantage in doing this. Other than it would make the npm scripts shorter by removing some characters.